views/js/qtiCreator/widgets/interactions/matchInteraction/states/Map.js

```diff
@@ -19,11 +19,6 @@ define([
         //finally, apply defined correct response and response mapping:
         responseWidget.setResponse(interaction, _.values(response.getCorrect()));
 
-        matchInteractionArea
-            .prop('disabled', true)
-            .prop('checked', false)
-            .addClass('disabled');
-
```

Comment:
Any idea why it was here in the first place? :)

Reply:
No idea...  that's why I was hesitant to delete... maybe @habemuscode knows better as it seems that last changes were made by him last year in [this commit](https://github.com/oat-sa/extension-tao-itemqti/commit/debdf3966842b20e3db41348be0ca0bc4e945820#diff-de84230cfd9c4abac34e1eb867bdce7c17b72965fb6640a14af56950b3345a4d) ?